clang/include/clang/Basic/AttrDocs.td

```diff
@@ -1473,6 +1473,66 @@ Further reading for other annotations:
   }];
 }
 
+def CHERIMMIODeviceDocs : Documentation {
```

Comment:
Maybe `CHERIOTMMIO` rather than `CHERIMMIO`?

Reply:
[c36946c](https://github.com/CHERIoT-Platform/llvm-project/pull/147/commits/c36946cbf1f9989c9fd5c17ce1104fa548cc7c87) renames the attributes (and the respective docs) to `CHERIOTMMIO...` and `CHERIOTSharedObject...`.